.ci/Jenkinsfile_cloud

```
@@ -19,51 +21,44 @@ pipeline {
-                                    for (def attempt = 1; attempt <= 3; attempt++) {
-                                        try {
-                                            withVaultSecret(secret: 'secret/kibana-issues/dev/cloud-staging-api-key', secret_field: 'value', variable_name: 'API_KEY') {
-                                                sh """./kibana-load-testing/scripts/deploy_and_test.sh -v '${params.STACK_VERSION}' -c '${params.DEPLOY_CONFIG}' -s '${params.SIMULATION}'"""
-                                            }
-                                            loadPipeline.uploadGatlingReport(i * j)
-                                            if (params.INGEST_RESULTS.toBoolean()) {
-                                                withVaultSecret(secret: 'secret/kibana-issues/prod/coverage/elasticsearch', secret_field: 'host', variable_name: 'HOST_FROM_VAULT') {
-                                                    withVaultSecret(secret: 'secret/kibana-issues/prod/coverage/elasticsearch', secret_field: 'username', variable_name: 'USER_FROM_VAULT') {
-                                                        withVaultSecret(secret: 'secret/kibana-issues/prod/coverage/elasticsearch', secret_field: 'password', variable_name: 'PASS_FROM_VAULT') {
-                                                            sh '''./kibana-load-testing/scripts/ingest_results.sh'''
-                                                        }
-                                                    }
-                                                }
-                                            }
-                                            break
-                                        } catch (Exception e) {
-                                            if (attempt == 3) {
-                                                throw new Exception("Could not succeed after ${attempt} attempts")
```

My guess it was added to retry cloud deployment specifically. But it looks like when it fails, it fails even with the retry logic, so it is better to fail fast and communicate with Cloud team

---

Yeah I agree this is probably unnecessary now with using a single deployment 👍